connection/BUILD

```diff
@@ -46,6 +46,7 @@ java_library(
         "@maven//:io_grpc_grpc_netty",
         "@maven//:io_grpc_grpc_stub",
         "@maven//:io_netty_netty_all",
+        "@maven//:io_netty_netty_handler",
```

Comment:
should it be under `deps` or `runtime_deps`?

Reply:
`deps`! It's needed at build time:
```
ERROR: /Users/james/Repos/typedb-client-java/connection/BUILD:26:13: Building connection/libconnection.jar (15 source files) failed: (Exit 1): java failed: error executing command external/remotejdk11_macos/bin/java -XX:-CompactStrings '--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED' ... (remaining 17 arguments skipped)
connection/cluster/ClusterServerClient.java:68: error: [strict] Using type io.netty.handler.ssl.SslContext from an indirect dependency (TOOL_INFO: "external/maven/v1/https/repo1.maven.org/maven2/io/netty/netty-handler/4.1.87.Final/netty-handler-4.1.87.Final.jar").
                SslContext sslContext;
                ^
```